modules/plugins/cache-compat.php

```diff
@@ -36,7 +36,7 @@ public function add_cookie_script() {
 			'(function() {
 				var expirationDate = new Date();
 				expirationDate.setTime( expirationDate.getTime() + %d * 1000 );
-				document.cookie = "%s=%s; expires=" + expirationDate.toUTCString() + "; path=%s%s";
+				document.cookie = "%s=%s; expires=" + expirationDate.toUTCString() + "; path=%s%s; HttpOnly";
```

Comment:
I don't think this will work - according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#JavaScript_access_using_Document.cookie):

> Cookies created via JavaScript cannot include the `HttpOnly` flag.

Would be little weird if it had worked when the point of `HttpOnly` flag is that you cannot access cookies via `document.cookie`.

Reply:
To be honest, I was not sure about it and must have skipped the sentence during research. I thought it might work as a write-only operation. I will remove it.

Reply:
You still have the problem remaining that this snippet will stop working at all with cookies that are HTTP only :-) I don't know if Polylang authors are willing to sacrifice this piece of functionality.

Reply:
You mean that a cookie set this way is not protected? True, but at least the default cookie is protected. I do not know about a way to fix it and I would leave it this way for now.

Reply:
Sorry, disregard my last comment. I somehow assumed that cookie value needs to be readable in JS too, but this is not the case.